core/src/main/resources/hudson/PluginWrapper/uninstall.groovy

```diff
@@ -18,7 +18,7 @@ l.layout(permission: Jenkins.ADMINISTER) {
         h1 {
             text(title)
         }
-        p { raw _("msg", my.displayName) }
+        p { raw _("msg", my.displayName, my.url) }
```

Comment:
Is a URL present in all places? E.g locally built plugins? / private ones?

Reply:
Locally build ones hosted on jenkinsci yes, private or proprietary plugins would depend on the URL provided I guess.

Reply:
docs say it can be `null`:
https://github.com/jenkinsci/jenkins/blob/6b5c21debd8430a662fed11e232a0ddf614edece/core/src/main/java/hudson/PluginWrapper.java#L600-L601

should probably handle that by displaying different messaging / removing the link